### DISCUSSION TITLE
2 Jingles played one after another as a result of skipping songs

### DISCUSSION THREAD

**Discussion by unudelaradio** [AzuraCast/AzuraCast#5582] [Q&A]
Hello, 

Am i the only one who noticed this problem? I have a simple configuration, with 2 playlists, Music and Jingles. But sometimes the AutoDJ plays 2 jingles one after another and i don't understand why?

The AutoDJ should play only one jingle, once per 1 Songs. But sometimes, it loads 2 jingles, one after another. 

Later Edit:

It skips songs. And the result is playing 2 jingles, one after another. Now the question is: why skipping songs? I manually added the skipped song in queue and there's no problem with the song/media file. It plays!

Here's a screenshot:
![skiped-song](https://user-images.githubusercontent.com/74244061/178093453-7e105ea6-4b9e-4f1c-9ce4-85872bc97399.png)

**Comment by atmix** [comment]
Same thing with me. Plays two jingles in a row from time to time. How to fix it?

**Comment by unudelaradio** [comment]
After few hours of monitoring the Song Queue the problem is clear to me:
1. When Jingles are set to play Once per 1 song, for some reasons, some songs from general rotation are skipped, and the result is hearing jingle after jingle.
2. When Jingles are set to play Once per 2 songs, everything seems to work fine. No skipped songs.

Setting Jingles to play once per 2 songs would be a solution for the moment. To avoid the repetition.

**Reply by lovefmhits** [reply]
hello
I too have double jingles and haven't found the problem yet

**Comment by unudelaradio** [comment]
The problem dissapears when Jingles are set to play once per 2 songs. So i tought a trick will work: i splitted my jingles in 2 separate playlists: Jingle Pack 1 and Jingle Pack 2, both set to play once per 2 songs. Then i rearranged the items in the Song Queue to make sure the order is the one i want: Jingles Pack 1, Music, Jingle Pack 2, Music.

Unfortunately the trick didn't work. Seems that music is skipped again from time to time. Conclusion is: when music is caught between 2 jingle mode playlists, it might be skipped.

**Comment by radijacija** [comment]
I have similar problems. Songs skipping and player stuck with displaying one single song played previously.  It's annoying

**Comment by DJMic** [comment]
I have not this problem. I also set my playlist Jingles to play Once per 1 song. Which Azuracast version are you using ?

**Reply by radijacija** [reply]
not the jingles. it is skipping 2-5 songs for no obvious reason
until i solve this problem, i il not go further
i am loging in through asurahosting, not directly. it should be updated fully?
